### DISCUSSION TITLE
What's the deal with the monorepo?

### DISCUSSION THREAD

**Discussion by glaprida** [GrapesJS/grapesjs#6113] [General]
I see you guys are moving this to a monorepo. 
What does that even mean? Is the repository gonna change?

Sorry for the ignorance but do we (those of us who use grapes js in our projects) need to do anything about it? 

Thanks!

**Comment by artf** [comment]
No, at the moment, nothing gonna change but we're aiming to transform grapesjs in a more headless framework, so all the UI-related code will be outside of it.
One day there will be a breaking change where the core will be published on `@grapesjs/core` and the rest inside its own package (eg. `@grapesjs/core-ui`). That means when you'll need to run a headless grapesjs instance (eg. in a web worker, nodejs, etc.) you won't load UI related code (right now it's not executed but it's there)
